app/controllers/form_controller.rb

```diff
@@ -45,11 +46,15 @@ def review
   end
 
   def show_page
+    if request.params["referrer"] == "interruption_screen"
+      @interruption_page_id = request.headers["HTTP_REFERER"].split("/")[-1].split("?")[0].underscore
```

Comment:
I thought rails would have helpers that would go a bit further than this in terms of making it more readable
```suggestion
      @interruption_page_id = URI.parse(request.headers["HTTP_REFERER"]).path.split("/").last.underscore
```

Reply:
Yup! Way nicer, thanks